addons/test_mail/tests/test_mail_mail.py

```diff
@@ -790,25 +790,6 @@ def test_mail_mail_values_email_multi(self):
             [['test.cc.1@test.example.com', 'test.cc.2@test.example.com']] * 3,
         )
 
-    def test_mail_mail_values_headers(self):
```

Comment:
Shouldn't we keep the test at least for return-path ?

Reply:
I think that's covered by `test_mail_mail_return_path` already